rustls/build.rs

```diff
@@ -0,0 +1,12 @@
+/// This build script allows us to enable the `read_buf` language feature only
+/// for Rust Nightly.
+///
+/// See the comment in lib.rs to understand why we need this.
+
+#[rustversion::not(nightly)]
+fn main() {}
+
+#[rustversion::nightly]
```

Comment:
Would it work to stick the attribute on the `println!()` statement directly, avoiding the repetition here?

Reply:
> Would it work to stick the attribute on the `println!()` statement directly, avoiding the repetition here?

No, I tried:
```
error[E0658]: custom attributes cannot be applied to statements
  --> rustls\build.rs:10:5
   |
10 |     #[rustversion::nightly]
   |     ^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #54727 <https://github.com/rust-lang/rust/issues/54727> for more information
   = help: add `#![feature(proc_macro_hygiene)]` to the crate attributes to enable

For more information about this error, try `rustc --explain E0658`.
```